File: codingTest/baekjoon/javascript/2480.js
var inp = require("fs").readFileSync("/dev/stdin").toString().split("\n");

var [a, b, c] = inp[0].split(" ");
if (a == b && b == c) console.log(10000 + a * 1000);
else if (a == b || a == c) console.log(1000 + a * 100);
else if (b == c) console.log(1000 + b * 100);
else console.log(Math.max(a, b, c) * 100);

// var rl = require("readline").createInterface({ input: process.stdin, output: process.stdout });

// var input = [];
// rl.on("line", line => {
//   input.push(line);
// }).on("close", () => {
//   var [a, b, c] = input[0].split(" ");
//   if (a == b && b == c) console.log(10000 + a * 1000);
//   else if (a == b || a == c) console.log(1000 + a * 100);
//   else if (b == c) console.log(1000 + b * 100);
//   else console.log(Math.max(a, b, c) * 100);
// });

File: codingTest/baekjoon/python/2480.py
# 1. 모두 같음 -> 10000 + 눈 * 1000
# 2. 2개만 같음 -> 1000 + 눈 * 100
# 3. 모두 다름 -> 가장 큰 눈 * 100
a, b, c = map(int, input().split())
if a == b == c:
    print(10000 + (a * 1000))
elif a == b or a == c:
    print(1000 + (a * 100))
elif b == c:
    print(1000 + (b * 100))
else:
    print(max(a, b, c) * 100)

# set 사용하기
# l = sorted(list(map(int, input().split())))
# s = set(l)
# if len(s) == 1:
#     print(10000 + (l[0] * 1000))
# elif len(s) == 2:
#     print(1000 + (l[1] * 100))
# else:
#     print(l[2] * 100)